app/src/main/java/io/homeassistant/companion/android/onboarding/authentication/AuthenticationFragment.kt

```diff
@@ -78,6 +78,7 @@ class AuthenticationFragment : Fragment() {
                                     error: WebResourceError?
                                 ) {
                                     super.onReceivedError(view, request, error)
+                                    Log.e(TAG, "onReceivedError: $error")
```

Comment:
This will result in a fairly useless error message like:
```
2022-07-27 18:13:26.538 4100-4100/io.homeassistant.companion.android.debug E/AuthenticationFragment: onReceivedError: pE0@93b3f73
```
It should use the `getErrorCode()` and `getDescription()` functions to provide something usable.